docker-compose.yml

```diff
@@ -1,14 +1,16 @@
-version: '3'
+version: '3.5'
 services:
-  node:
-    image: node:18-alpine
-    user: node:node
-    working_dir: /var/www/app
+  app:
+    build:
+      context: ./
+      dockerfile: Dockerfile-dev
+    env_file:
+      - .env
     volumes:
-      - .:/var/www/app
-    command: ['sh', '-c', 'yarn --production=false --frozen-lockfile && npx --yes concurrently "yarn start:storybook" "yarn start:dev"']
```

Comment:
maybe add a note in the readme to explain how to launch storybook that will not be started automatically anymore